shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1197,6 +1210,8 @@ void WebContents::CloseContents(content::WebContents* source) {
 
   for (ExtendedWebContentsObserver& observer : observers_)
     observer.OnCloseContents();
+
+  Destroy();
```

Comment:
If we're destroying this now and weren't before, would we want to consider that breaking? or are the mechanics of this such that users would experience no change?

Reply:
I don't believe there's any functional difference here; previously the WebContents was destroyed by the BrowserWindow in `OnCloseContents`, now it's closed here instead.